rules-java/api/src/main/resources/reports/templates/ignored_files.ftl

```diff
@@ -15,6 +15,9 @@
 
 <#macro ignoredFileRenderer reportModel>
     <div class="panel panel-primary">
+        <div class="panel-heading">
+            <h3 class="panel-title">Ignored files</h3>
```

Comment:
Shouldn't "files" be upper-case ("Files")? I think that it may be wrong in a few places in this report.

Reply:
right, I didn't catch this.

Reply:
Good catch, i missed this too :(